### DISCUSSION TITLE
Support copy-paste formatted text and/or Markdown symbols to Discord

### DISCUSSION THREAD

**Discussion by hsandt** [zim-desktop-wiki/zim-desktop-wiki#2510] [Ideas]
Currently, there is no way to copy-paste text containing formatting such as italics and bold to Discord text input fields.

For instance, both a string with actual **bold** characters and a string with actual \*\*surrounding asterisks\*\* will be copied and pasted to Discord as normal text.

While I can understand that Zim bold/italics is lost as it doesn't use Markdown under the hood, I don't understand how actual asterisk characters can get lost in the process. If I use a raw text editor like VS Code and copy text with \*\*surrounding asterisks\*\*, it will properly paste them, so the final text will appear bold in Discord.

Ultimately, it would be nice if both cases were supported.

**Comment by introt** [comment]
Related: #913
